recipes/recipes_emscripten/xeus-python/recipe.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
   - python
   - libpython
   - nlohmann_json
+  - nlohmann_json-abi
   - xeus =5.2.2
   - xeus-lite =3.2.1
```

Comment:
Probably the xeus + xeus-lite pin needs to be removed too (the next build after the pinned version started promoting a shared build of xeus -> which was then propagated to xeus-python too https://github.com/jupyter-xeus/xeus-python/pull/657 and now with this latest release we should be able to use the shared xeus build)